Test/VimCoreTest/InsertModeIntegrationTest.cs

```diff
@@ -2655,7 +2693,7 @@ public void WordCompletion_Escape_Async()
                 Dispatcher.DoEvents();
                 _vimBuffer.Process(KeyNotationUtil.StringToKeyInput("<Esc>"));
                 Assert.Equal(ModeKind.Normal, _vimBuffer.ModeKind);
-                Assert.Equal(0, _textView.GetCaretPoint().Position);
+                Assert.Equal(2, _textView.GetCaretPoint().Position);

```

Comment:
Why is `2` correct here? I can understand why `1` would be correct here but not `2` as it moves beyond the original word.

Reply:
My testing of Vim completes the whole word `cat` and escape exits insert mode thus moving the caret back to `2`, so `2` is the Vim behavior. This would be clearer if the test also asserted the contents of the line after the operation.